framework/src/controller/config/dirs.js

```diff
@@ -0,0 +1,10 @@
+const rootDir = process.cwd();
+
+const systemDirs = appLabel => ({
+	root: rootDir,
+	temp: `${rootDir}/tmp/${appLabel}/`,
+	sockets: `${rootDir}/tmp/${appLabel}/sockets`,
```

Comment:
Will the application create those directories if they do not exist?

Reply:
Yes